pages/index.tsx

```diff
@@ -1,9 +1,34 @@
 import type { NextPage } from "next";
 import { ThemeProvider } from "@mui/material";
 import { theme } from "../styles/theme";
+import { Divider } from "@mui/material";
```

Comment:
For all the mui imports, you can import them together! So it'd be like import { ThemeProvider, Divider, Typography, Link } from "@mui/material";